server/datastore/mysql/migrations/tables/20241002210000_PolicyAutomationRunScript.go

```diff
@@ -0,0 +1,26 @@
+package tables
```

Comment:
missing migration test?

Reply:
Added in [7c5bf4d](https://github.com/fleetdm/fleet/pull/22472/commits/7c5bf4d88aff1e96b8fce6f98ebeb65482dea39b)